src/forms/textInput.test.jsx

```diff
@@ -1,129 +1,120 @@
 import React from 'react';
-import TestUtils from 'react-addons-test-utils';
-import { TestWrapper } from '../utils/testUtils';
 import TextInput from './TextInput';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
 
 describe('TextInput', function() {
 	const onChange = jest.fn();
+
 	const LABEL_TEXT = 'Super Hero',
 		VALUE = 'Batman',
 		NAME_ATTR = 'superhero',
-		MAX_LEN = '20',
+		MAX_LEN = 20,
 		ERROR_TEXT = 'Too wimpy.';
 
-	let textInputComponent,
-		inputEl;
-
-	beforeEach(() => {
-		const formAttrs = {
-			id: NAME_ATTR,
-			maxLength: MAX_LEN,
-			error: ERROR_TEXT,
-			required: true,
-		};
-		textInputComponent = TestUtils.renderIntoDocument(
-			<TestWrapper>
+	const formAttrs = {
+		id: NAME_ATTR,
+		maxLength: MAX_LEN,
+		required: true,
+		error: ERROR_TEXT,
+	};
+
+	describe('snapshot check', () => {
+		it('matches snapshot', () => {
```

Comment:
probably better to be more specific about the inputs used to make the snapshot - it might 'match the snapshot' when given certain props, but not match the snapshot when given other props, so there's no general "matches snapshot" guarantee.